restricted_site_access.php

```diff
@@ -16,6 +16,15 @@
 // Try and include our autoloader.
 if ( is_readable( __DIR__ . '/vendor/autoload.php' ) ) {
 	require_once __DIR__ . '/vendor/autoload.php';
+
+	$compat_checker = new \WP_Compat_Checker\WP_Compat_Checker();
```

Comment:
I know this isn't a review of the compat checker code but wondering if it would be better to either have a different namespace or different class name, so we don't just duplicate those in code like this?

Reply:
We can do that. I came up with these names just so that I can begin with experimenting this approach. Do you have any suggestions?